src/main/java/com/mateacademy/springmvc/bpp/UserBeanPostProcessor.java

```diff
@@ -0,0 +1,29 @@
+package com.mateacademy.springmvc.bpp;
+
+import com.mateacademy.springmvc.entity.User;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.config.BeanPostProcessor;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserBeanPostProcessor implements BeanPostProcessor {
+    private static final Logger LOGGER = LoggerFactory.getLogger(UserBeanPostProcessor.class);
```

Comment:
Minor: If you already use Lombok why not use for a logger?
https://projectlombok.org/features/log